### DISCUSSION TITLE
'POS tagging' output is not correct

### DISCUSSION THREAD

**Discussion by adityakadrekar16** [explosion/spaCy#13310] [Help: Other Questions]
## How to reproduce the behaviour

Hi, I tried the 4 english pipelines (en_core_web - md, sm, lg, trf) for POS tagging. I know that spacy is case sensitive but how are words like 'Water', 'Wheat', 'Cereal', 'Dry', 'Information', 'Research', etc tagged as Proper Noun (NNP, PROPN)?
These words are title case.  Even lowercase words like oil, dry, nutritional, law, express are tagged as NNP.

I thought maybe 'md' model is not large enough to recognize it but even large models like 'lg' and 'trf' are giving poor results. Am I doing something wrong? Can you please help me?

import spacy
nlp_trf = spacy.load("en_core_web_trf")
doc = nlp_trf(text)
for token in doc:
    print(token.text, token.lemma_, token.tag_, token.pos_)

Thanks,
Aditya

## Your Environment

* Operating System: Mac M1 Pro Venture 13.6
* Python Version Used: 3.10.13
* spaCy Version Used: 3.6.1
* Environment Information:

**Comment by svlandeg** [comment]
Hi! The accuracy of the tagger is not 100%, so you will definitely find cases that are incorrect. Nevertheless, our [benchmarks](https://spacy.io/usage/facts-figures#benchmarks) put its accuracy at above 97% on the OntoNotes 5.0 corpus.

Are you giving it full grammatical sentences to tag? Can you give some examples of systematic errors of both the lowercase and uppercase words, where they are used and wrongly tagged in full sentences?

**Comment by adityakadrekar16** [comment]
Yes, I am giving full grammatical sentences to tag. I cannot share the sentences as it is confidential business data. But I have given examples above and here are few more examples of incorrect tagging:
''' Wheat NNP PROPN
Water NNP PROPN
energy NNP PROPN'
pea NNP PROPN
protein NNP PROPN
express NNP PROPN"

This is what I am printing above
`doc = nlp(pdf_text)
for token in doc:
    print(token.text, token.tag_, token.pos_)`
